docs/k8s-kubespray.md

```diff
@@ -140,3 +140,20 @@ ansible-playbook --inventory /etc/genestack/inventory/openstack-flex-inventory.y
     Given the use of a venv, when running with `sudo` be sure to use the full path and pass through your environment variables; `sudo -E /home/ubuntu/.venvs/genestack/bin/ansible-playbook`.
 
 Once the cluster is online, you can run `kubectl` to interact with the environment.
+
+## Installing Kubernetes
+
+Currently only the k8s provider kubespray is supported and included as submodule into the code base.
+A default inventory file for kubespray is provided at `/etc/genestack/inventory` and must be modified.
+Existing OpenStack Ansible inventory can be converted using the `/opt/genestack/scripts/convert_osa_inventory.py`
```

Comment:
I think the inventory convert process should be a callout as a tip.

Something like 

```suggestion
!!! tip

    Existing OpenStack Ansible inventory can be converted using the `/opt/genestack/scripts/convert_osa_inventory.py`
```